ko_KR/readme.md

```diff
@@ -0,0 +1,971 @@
+
+다음 예시를 봅시다, Node의 표준 [`assert` 라이브러리](https://nodejs.org/api/assert.html)를 사용합니다:
+
+```js
+const a = /foo/;
+const b = 'bar';
+const c = 'baz';
+require('assert').ok(a.test(b) || b === c);
+```
+
+만약 Node REPL에 붙여넣고 실행하면 다음의 결과를 반환합니다:
+
+```
+AssertionError: false == true
+```
+
+AVA에선, 다음을 테스트하면:
+
+```js
+test(t => {
+	const a = /foo/;
+	const b = 'bar';
+	const c = 'baz';
+	t.true(a.test(b) || b === c);
+});
+```
+
+다음의 결과가 반환됩니다:
+
+```
+t.true(a.test(b) || b === c)
+       |      |     |     |
+       |      "bar" "bar" "baz"
+       false
+```
+
+## 프로세스 고립
+
+각 테스트파일은 분리된 Node.js 프로세스에서 실행됩니다. 이것은 다른 파일에 영향을 주지 않고 전역 상태 또는 빌트-인을 변경할 수 있도록 합니다. 또한 이는 현대의 멀티-코어 프로세서에서 아주 좋은 성능을 발휘하며, 여러 테스트 파일을 병행으로 실행할 수 있도록 합니다.
+
+## 도움말
+
+### 임시 파일
+
+테스트를 동시에 실행하는 것은 약간 모험이 될 수 있습니다. 파일 IO 작업은 예시 중 하나입니다.
+
+보통, 직렬 테스트는 현재 테스트 경로에 임시 폴더를 하나 생성하고 테스트가 끝나면 그 폴더를 지웁니다. 이는 테스트를 동시에 실행할 때 다른 테스트와 충돌이 일어나는 등의 이유로 인해 작동하지 않을 것입니다. 가장 알맞은 방법은 각 테스트에서 새로운 임시 경로를 생성하여 사용하는 것입니다. [`tempfile`](https://github.com/sindresorhus/tempfile)과 [`temp-write`](https://github.com/sindresorhus/temp-write) 모듈이 도움이 될 것입니다.
+
+### 디버깅
+
+AVA는 기본적으로 테스트를 동시에 실행하며, 디버깅할 때 최적의 환경을 제공합니다. 대신에, 테스트를 직렬로 실행하려면 `--serial` 옵션을 사용하면 됩니다:
+
+```console
+$ ava --serial
+```
+
+### 코드 커버리지
+
+AVA는 [테스트 파일을 spawn](#프로세스-고립)하기 때문에 [`istanbul`](https://github.com/gotwarlost/istanbul)을 코드 커버리지로 사용할 수 없습니다. 대신에 [`nyc`](https://github.com/bcoe/nyc)를 사용할 수 있으며, 사실상 서브 프로세스를 지원하는 `istanbul`입니다.
+
+버전 `5.0.0`부터 트랜스파일에 관계없이 실제 코드의 커버리지를 보고하기 위해 소스 맵을 사용합니다. 테스트 대상 코드가 인라인 소스 맵 또는 소스 맵 파일에 대한 참조 중 하나를 포함하는지 확인하세요. `babel/register`를 사용하고 있다면 `.babelrc`에서 `sourceMaps` 옵션을 `inline` 으로 설정할 수 있습니다.
+
+## FAQ
+
+### 왜 `mocha`, `tape`, `tap`을 안 쓰나요?
+
+Mocha는 기본적인 인터페이스로써 `describe`와 `it` 같은 암시적인 전역 변수를 사용하도록 요구합니다. (많은 사람이 사용합니다) 이는 좋지 않은 것을 고집하며 프로세스 고립 없이 테스트를 직렬로 실행하므로 느립니다.
+
+Tape와 tap은 상당히 괜찮습니다. AVA는 이들의 문법에서 많은 영감을 받았습니다. 하지만 여전히 테스트를 직렬로 실행합니다. 이들의 기본 [TAP](https://testanything.org) 출력은 사용자 친화적이지 않고 이러한 이유로 결국 언제나 외부 tap 리포터를 사용해야 합니다.
+
+이와 대조하여 AVA는 좋은 것을 고집하며 각 테스트 파일에 분리된 프로세스와 함께 테스트를 동시에 실행합니다. 또한 기본 리포터는 보기 좋으며, 여전히 AVA는 CLI 플래그를 통해 TAP 출력을 지원합니다.
+
+### 커스텀 리포터는 어떻게 사용하나요?
+
+AVA는 TAP 포맷을 지원하며 따라서 모든 [TAP 리포터](https://github.com/sindresorhus/awesome-tap#reporters)와 호환 가능합니다. [`--tap` 플래그](#선택적인-tap-출력)를 사용하여 TAP 출력을 활성화 하세요.
+
+### 이 프로젝트 이름은 어떻게 쓰고 발음이 어떻게 되나요?
+
+AVA입니다. Ava나 ava가 아닙니다. [`/ˈeɪvə/` ay-və](https://github.com/sindresorhus/ava/blob/master/media/pronunciation.m4a?raw=true)로 발음됩니다.
+
+### 헤더의 배경은 무엇인가요?
+
+[안드로메다 은하](https://ko.wikipedia.org/wiki/%EC%95%88%EB%93%9C%EB%A1%9C%EB%A9%94%EB%8B%A4_%EC%9D%80%ED%95%98)입니다.
+
+### 동시성과 병행성은 무슨 차이가 있나요?
+
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
+- [엔드 포인트 테스팅](docs/recipes/endpoint-testing.md)
+- [`t.plan()`를 언제 사용할까](docs/recipes/when-to-use-plan.md)
```

Comment:
Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/when-to-use-plan.md)`